strongbox-cron/strongbox-cron-tasks/pom.xml

```diff
@@ -106,82 +78,6 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
-
```

Comment:
Are we really dropping all this crap? :laughing: 
I can't believe I lived to see the day we don't need all this sourcery!

Thanks! :)

Reply:
well, I just moved it to [strongbox-parent](https://github.com/strongbox/strongbox-parent/pull/30/files#diff-600376dffeb79835ede4a0b285078036R240) :smiley: 
to avoid copypaste configuration in all our modules